clidocstool_yaml_test.go

```diff
@@ -42,12 +39,10 @@ func TestGenYamlTree(t *testing.T) {
 	for _, tt := range []string{"docker_buildx.yaml", "docker_buildx_build.yaml", "docker_buildx_stop.yaml"} {
 		tt := tt
 		t.Run(tt, func(t *testing.T) {
-			fres := filepath.Join(tmpdir, tt)
-			require.FileExists(t, fres)
-			bres, err := ioutil.ReadFile(fres)
+			bres, err := os.ReadFile(filepath.Join(tmpdir, tt))
```

Comment:
The `require.FileExists()` check felt a bit redundant here, as `os.ReadFile()` will already return an error if the file doesn't exist (which would also include the name of the missing file.

Reply:
Yes indeed